src/components/signin.jsx

```diff
@@ -1,54 +1,54 @@
-import React, { Component } from "react";
-
-
+import React from "react";
+import { Link } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.css";
+import "../index.css";
 
 const Signin = () => {
-    return ( 
-        <div class="container">
-        <h1 class="row justify-content-center mt-5">ESTATE CITADEL</h1>
-        <div class="row justify-content-center mt-5">
-          <div class="col-md-6">
-            <div class="card">
-              <div class="card-body">
-                <img
-                  class="col-md-"
-                  src="/images/logo.png"
-                  alt="Logo"
-                  width="250"
-                  height="100"
-                />
-                <h3 class="card-title text-center">Signin</h3>
-                <form>
-                  <div class="form-group">
-                    <label for="email">Email/Username</label>
-                    <input
-                      type="text"
-                      class="form-control"
-                      id="email"
-                      placeholder="Enter email or username"
-                      required
-                    />
-                  </div>
-                  <div class="form-group">
-                    <label for="password">Password</label>
-                    <input
-                      type="password"
-                      class="form-control"
-                      id="password"
-                      placeholder="Enter password"
-                      required
-                    />
-                  </div>
-                  <button type="submit" class="btn btn-primary">
-                    Signin
-                  </button>
-                </form>
-              </div>
+  return (
+    <body>
+      <div className="container">
+        <h1 className="header">ESTATE CITADEL</h1>
+        <div className="form">
+          <img
+            className="logo"
+            src="/images/logo.png"
```

Comment:
It's better to import your images above, before using them in the code. It facilitates things if you want to change and use a different image later.